service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/UploadAdditionalApplicationsController.java

```diff
@@ -84,10 +84,13 @@ public AboutToStartOrSubmitCallbackResponse handleMidEvent(@RequestBody Callback
 
         caseDetails.getData().putAll(applicationsFeeCalculator.calculateFee(caseData));
 
-        if (featureToggleService.isServeOrdersAndDocsToOthersEnabled() && isNotEmpty(caseData.getAllOthers())) {
+        if (featureToggleService.isServeOrdersAndDocsToOthersEnabled()
+            && (isNotEmpty(caseData.getAllOthers()) || isNotEmpty(caseData.getRespondents1()))) {
             caseDetails.getData().put("hasOthers", "Yes");
-            caseDetails.getData().put("others_label", othersService.getOthersLabel(caseData.getAllOthers()));
-            caseDetails.getData().put("othersSelector", newSelector(caseData.getAllOthers().size()));
+            caseDetails.getData().put("others_label", peopleInCaseService.buildPeopleInCaseLabel(
+                caseData.getAllRespondents(), caseData.getOthers()));
+            caseDetails.getData().put("othersSelector", newSelector(
+                caseData.getAllRespondents().size() + caseData.getAllOthers().size()));
```

Comment:
Maybe it would be a good idea to extract this into a variable? (caseData.getAllRespondents().size() + caseData.getAllOthers().size())